tsl/src/chunk.c

```diff
@@ -218,19 +215,17 @@ chunk_invoke_drop_chunks(Oid relid, Datum older_than, Datum older_than_type)
 		makeNullConst(INT8OID, -1, InvalidOid),
 		castNode(Const, makeBoolConst(false, true)),
 	};
-
 	char *const schema_name = ts_extension_schema_name();
-	char *const function_name = DROP_CHUNKS_FUNCNAME;
 	List *fqn = list_make2(makeString(schema_name), makeString(function_name));
-	funclist = FuncnameGetCandidates(fqn, DROP_CHUNKS_NARGS, NIL, false, false, false);
+	funclist = FuncnameGetCandidates(fqn, lengthof(argarr), NIL, false, false, false);
 
-	if (funclist->next != NULL)
+	if (!funclist || funclist->next != NULL)
 		elog(ERROR, "could not find drop_chunks function");
 
```

Comment:
should use LookupFuncName here

Reply:
LookupFuncName allows you to pass Oid[] for the args so you dont need to construct Consts which probably allows you to remove 10 lines of code

Reply:
Ah, that's what you meant. Sounds useful, will take a look.

Reply:
well i guess you still need that for the actual call

Reply:
Yes, just realized that.

Reply:
hmmm i would still prefer LookupFuncName here cause that would validate argument datatypes

Reply:
I second using LookupFuncName